pkg/metrics/http/server_metrics.go

```diff
@@ -29,7 +34,7 @@ func NewServerMetrics(opts ...ServerMetricsOption) *ServerMetrics {
 			config.counterOpts.apply(prometheus.CounterOpts{
 				Name: "http_server_received_total_requests",
 				Help: "Tracks the total number of HTTP requests.",
-			}), []string{"handler_name", "method", "code"}),
+			}), []string{"handler_name", "method", "status_code", "error_code"}),
```

Comment:
`status_code`-->`http_code`?
`error_code`-->`interval_error_code`?

Reply:
Is it too long?

Reply:
fixed